lib/mixlib/shellout/windows.rb

```diff
@@ -122,7 +122,7 @@ def run_command
                     wmi = WmiLite::Wmi.new
                     kill_process_tree(process.process_id, wmi, logger)
                     Process.kill(:KILL, process.process_id)
-                  rescue
+                  rescue Errno::EIO
```

Comment:
We still not catching all required errors which was there earlier https://github.com/chef/mixlib-shellout/pull/157/files#diff-a49639dc02ce0578d8000ee0aecf5210. If we done this just to avoid rubocop issues we should make sure we are not having regression or doing something wrong to avoid rubocop issues.